src/vs/workbench/services/files/node/fileService.ts

```diff
@@ -28,10 +28,16 @@ import pfs = require('vs/base/node/pfs');
 import encoding = require('vs/base/node/encoding');
```

Comment:
Pending response in https://github.com/Microsoft/vscode/pull/12400#pullrequestreview-4093616